Run: 28
/cm/local/apps/slurm/var/spool/job990658/slurm_script: line 26: 
pip install -e tpot2
pip install -r tpot2/ImputerExperiments/requirements_.txt
: No such file or directory
RunStart
Version 0.1.7a0 of tpot2 is outdated. Version 0.1.8a0 was released Tuesday September 17, 2024.
starting loops
../data/r/42712/42712.pkl
       season  year  month  hour  holiday  weekday  workingday weather   temp  \
0      spring     0      1     0    False        6       False   clear   9.84   
1      spring     0      1     1    False        6       False   clear   9.02   
2      spring     0      1     2    False        6       False   clear   9.02   
3      spring     0      1     3    False        6       False   clear   9.84   
4      spring     0      1     4    False        6       False   clear   9.84   
...       ...   ...    ...   ...      ...      ...         ...     ...    ...   
17374  spring     1     12    19    False        1        True   misty  10.66   
17375  spring     1     12    20    False        1        True   misty  10.66   
17376  spring     1     12    21    False        1        True   clear  10.66   
17377  spring     1     12    22    False        1        True   clear  10.66   
17378  spring     1     12    23    False        1        True   clear  10.66   

       feel_temp  humidity  windspeed  
0         14.395      0.81     0.0000  
1         13.635      0.80     0.0000  
2         13.635      0.80     0.0000  
3         14.395      0.75     0.0000  
4         14.395      0.75     0.0000  
...          ...       ...        ...  
17374     12.880      0.60    11.0014  
17375     12.880      0.60    11.0014  
17376     12.880      0.60    11.0014  
17377     13.635      0.56     8.9981  
17378     13.635      0.65     8.9981  

[17379 rows x 12 columns]
<class 'pandas.core.frame.DataFrame'>
       season  year  month  hour  holiday  weekday  workingday weather   temp  \
17223  spring     1     12    12     True        2       False   clear  13.12   
3455   summer     0      5    21    False        6       False   clear  27.06   
12200  summer     1      5    13     True        1       False   clear  32.80   
4725     fall     0      7    19    False        3        True   clear  33.62   
14373    fall     1      8     2    False        1        True   clear  25.42   
...       ...   ...    ...   ...      ...      ...         ...     ...    ...   
13829    fall     1      8    10    False        6       False   clear  33.62   
6530   winter     0     10    16    False        2        True   clear  23.78   
8418   spring     0     12    10    False        4        True   misty  16.40   
2124   summer     0      4     9    False        0       False   clear  14.76   
15420  winter     1     10    17    False        2        True   misty  21.32   

       feel_temp  humidity  windspeed  
17223     16.665      0.76     6.0032  
3455      31.060      0.74    19.9995  
12200     39.395      0.63    19.0012  
4725      42.425      0.67    11.0014  
14373     28.030      0.88     8.9981  
...          ...       ...        ...  
13829     38.635      0.57    12.9980  
6530      27.275      0.56    22.0028  
8418      20.455      0.82     6.0032  
2124      15.910      0.50    32.9975  
15420     25.000      0.68    11.0014  

[13903 rows x 12 columns]
<class 'pandas.core.frame.DataFrame'>
Traceback (most recent call last):
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 591, in astype
    new_cats = new_cats.astype(dtype=dtype, copy=copy)
ValueError: could not convert string to float: 'fall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/main.py", line 166, in <module>
    main()
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/main.py", line 155, in main
    utils.loop_through_tasks(reg_experiments, regression_id_list, 
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/utils.py", line 143, in loop_through_tasks
    X_train, y_train, X_test, y_test = load_task(base_save_folder=save_folder, task_id=taskid, r_or_c= r_or_c)
  File "/common/ketrong/tpotexp/tpot2/ImputerExperiments/Locally_run_code/utils.py", line 109, in load_task
    preprocessing_pipeline.fit(X_train)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1473, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 469, in fit
    Xt = self._fit(X, y, routed_params)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 406, in _fit
    X, fitted_transformer = fit_transform_one_cached(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/joblib/memory.py", line 312, in __call__
    return self.func(*args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/pipeline.py", line 1310, in _fit_transform_one
    res = transformer.fit_transform(X, y, **params.get("fit_transform", {}))
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/utils/_set_output.py", line 313, in wrapped
    data_to_wrap = f(self, X, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1098, in fit_transform
    return self.fit(X, **fit_params).transform(X)
  File "/common/ketrong/tpotexp/tpot2/tpot2/builtin_modules/imputer.py", line 114, in fit
    self.imputer.fit(X[self.columns_], y)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 1473, in wrapper
    return fit_method(estimator, *args, **kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 421, in fit
    X = self._validate_input(X, in_fit=True)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 350, in _validate_input
    raise ve
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/impute/_base.py", line 332, in _validate_input
    X = self._validate_data(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/base.py", line 633, in _validate_data
    out = check_array(X, input_name="X", **check_params)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/sklearn/utils/validation.py", line 929, in check_array
    array = array.astype(new_dtype)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/generic.py", line 6643, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 430, in astype
    return self.apply(
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/managers.py", line 363, in apply
    applied = getattr(b, f)(**kwargs)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/internals/blocks.py", line 758, in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/dtypes/astype.py", line 237, in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/dtypes/astype.py", line 179, in astype_array
    values = values.astype(dtype, copy=copy)
  File "/common/ketrong/tpotexp/env/lib/python3.10/site-packages/pandas/core/arrays/categorical.py", line 602, in astype
    raise ValueError(msg)
ValueError: Cannot cast object dtype to float64
srun: error: esplhpc-cp020: task 0: Exited with exit code 1
